lldb/test/API/lang/swift/async/unwind/unwind_in_all_instructions/TestSwiftAsyncUnwindAllInstructions.py

```diff
@@ -0,0 +1,134 @@
+import lldb
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbtest as lldbtest
+import lldbsuite.test.lldbutil as lldbutil
+
+# This test creates 5 async frames:
+# * ASYNC___1___, which is called by
+# * ASYNC___2___, which is called by
+# * ...
+# * ASYNC___5___, which is called by
+#
+# The number of frames is important to exercise all possible unwind plans:
+# * The top frame (ASYNC___1___) is created by just inspecting the registers as
+# they are.
+# * The plan for ASYNC___1___ -> ASYNC___2___ is created through
+# `GetRuntimeUnwindPlan`, which is responsible for the transition from a real
+# frame to a virtual frame.
+# * The plan for ASYNC___2___ -> ASYNC___3___ is created through
+# GetFollowAsyncContextUnwindPlan, which is responsible to create a virtual
+# frame from another virtual frame.
+# * The plan for ASYNC___3___ -> ASYNC___4___ is created through
+# GetFollowAsyncContextUnwindPlan, but this time it follow the code path where
+# `is_indirect = True` (see its implementation).
```

Comment:
minor, but the `True` makes this look like a reference to some Python test code. `is_indirect = true` would fix that.